tests/scripts/thread-cert/Cert_6_1_01_RouterAttach.py

```diff
@@ -60,10 +70,55 @@ def test(self):
         self.nodes[LEADER].start()
         self.nodes[LEADER].set_state('leader')
         self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+        time.sleep(4)
```

Comment:
Since Line#71 has explicitly specify `leader` role, is the 4s sleep necessary here?
Or the first thread device would become leader automatically, how about removing Line#71 and adding 4s sleep before `assertEqual()` of Line#72 ?